framework/src/modules/chain/dpos/dpos.js

```diff
@@ -14,13 +14,21 @@
 
 'use strict';
 
-const { DelegatesList, EVENT_ROUND_FINISHED } = require('./delegates_list');
-const { DelegatesInfo } = require('./delegates_info');
+const { DelegatesList } = require('./delegates_list');
+const { DelegatesInfo, EVENT_ROUND_CHANGED } = require('./delegates_info');
```

Comment:
If you decide to use `events` object, need to destructure here 💃

Reply:
See answer above